expression/util.go

```diff
@@ -30,16 +30,30 @@ import (
 
 // ExtractColumns extracts all columns from an expression.
 func ExtractColumns(expr Expression) (cols []*Column) {
+	result := make([]*Column, 0, 5)
```

Comment:
Why choose 5?

Reply:
Nothing, just a constant like 42... I'm kidding...

For this:

```
result := make([]*Column, 0, 5)
```

Escape analysis in Go compiler can optimize the code make no allocation.
While if I write the code like this:

```
result := make([]*Column, 0, len(exprs))
```

The compiler optimization can't do the optimization, because `len(exprs)` is unknown.

You can try this code and view the different compiler output:

```
func f(len int) {
	x := make([]int, 0, 5)
       // x := make([]int, 0, len)
	use(x)
}
```

go tool compile -m test.go

```
test.go:6:11: main make([]int, 0, 5) does not escape
test.go:10:11: make([]int, 0, len) escapes to heap
```

Reply:
Why is it not 6 or 10 or something else?

Reply:
Please comment it.